patches/redux-auth-wrapper+2.1.0.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/node_modules/redux-auth-wrapper/helper/redirect.js b/node_modules/redux-auth-wrapper/helper/redirect.js
+index 0c4ed18..16eb362 100644
+--- a/node_modules/redux-auth-wrapper/helper/redirect.js
++++ b/node_modules/redux-auth-wrapper/helper/redirect.js
+@@ -103,7 +103,12 @@ exports.default = function (_ref) {
+         authenticatingSelector = allArgs.authenticatingSelector,
+         allowRedirectBack = allArgs.allowRedirectBack,
+         redirectAction = allArgs.redirectAction,
+-        redirectQueryParamName = allArgs.redirectQueryParamName;
++        redirectQueryParamName = allArgs.redirectQueryParamName,
++        context = allArgs.context;
++
++    if (!context) {
++      (0, _invariant2.default)(false, 'you must provide a custom context');
++    }
```

Comment:
we have type coverage, but added this since we use this method in some of our js files